src/winstanley/wdl.bnf

```diff
@@ -81,8 +81,8 @@ infix_operator ::= DOUBLE_PIPE | DOUBLE_AMPERSAND | DOUBLE_EQUAL | NOT_EQUAL | L
 function_call ::= IDENTIFIER LPAREN (expression (COMMA expression)*)? RPAREN
 array_or_member_access ::= IDENTIFIER array_or_member_access_suffix+
 array_or_member_access_suffix ::= LSQUARE expression RSQUARE | DOT IDENTIFIER
-object_literal ::= OBJECT LBRACE object_kv (COMMA object_kv)* RBRACE
-map_literal ::= LBRACE map_kv (COMMA map_kv)* RBRACE
+object_literal ::= OBJECT LBRACE object_kv (COMMA object_kv)* COMMA? RBRACE
+map_literal ::= LBRACE map_kv (COMMA map_kv)* COMMA? RBRACE
 array_literal ::= LSQUARE expression (COMMA expression)* RSQUARE
```

Comment:
Based on the [syntax for literal arrays, maps and objects](https://github.com/openwdl/wdl/blob/master/parsers/grammar.hgr#L433) in the Hermes grammar in the official WDL spec (the final grammar file you'll need to know about, and for now at least, the canonical one), arrays work the same as objects and maps.

Since we're here, we might as well do this for arrays too.